geometry_msgs/Point pos
geometry_msgs/Twist vel
geometry_msgs/Point[] neighbours_pos
geometry_msgs/Twist[] neighbours_vel
float64[] neighbours_rad
